dbms/src/Flash/Planner/PhysicalPlan.cpp

```diff
@@ -76,6 +76,17 @@ void PhysicalPlan::build(const tipb::DAGRequest * dag_request)
 void PhysicalPlan::buildTableScan(const String & executor_id, const tipb::Executor * executor)
 {
     TiDBTableScan table_scan(executor, executor_id, dagContext());
+    LOG_DEBUG(log, "pb table scan runtime filter size:{}", executor->tbl_scan().runtime_filter_list().size());
```

Comment:
What's the difference between runtime filter size in table_scan and pb table scan?

Reply:
There is no difference, it is just because I printed some more logs in order to debug a bug.